src/MoBi.Presentation/Presenter/EditFormulaPathListPresenter.cs

```diff
@@ -124,6 +124,9 @@ public void SetFormulaUsablePath(string newPath, FormulaUsablePathDTO dto)
          var path = new ObjectPath(newPath.ToPathArray());
          var formulaUsablePath = _formula.FormulaUsablePathBy(dto.Alias);
 
+         if (_circularReferenceChecker.HasCircularReference(path, _formulaOwner))
```

Comment:
I think we removed it because it takes way too much time on big projects
That's why it also became an option in the user settings.. Adding it back will create other issus but might be wrong

Reply:
Is that an option only when building a simulation or also when entering each alias?

Reply:
So, should I surround this with an option check or what do you think?

Reply:
I really don't know. I remember a time where EDITING of a fomula took forever in a project (from @PavelBal ) in fact
So we removed this But maybe it was due to the formula ?